mission_data/quiz_questions.yaml

```
@@ -0,0 +1,278 @@
+  - id: 0d0afdc3-d2ee-4024-bfa2-308bab5d5d8f
+    category_slug: policy
+    mission_slug: quiz-policy-intermediate-2
+    question: チームみらいのマニフェストの決め方で**誤っている**ものは？
+    option1: チームみらいのミッション・ビジョンやステップ1〜3の方向性に合致している提案だけを採択対象とする。
+    option2: 改善提案は AI で自動ラベリング・要約して整理する。
+    option3: 最終的な反映方針をチームでまとめる。
+    option4: チームが作成した反映方針を、AIが一件ずつ確認し、最終的な意思決定を行う。
+    correct_answer: 4
+    explanation: 最終的な意思決定は、党首である安野たかひろが一件ずつ確認し行うとしている。
+    question_order: 5
+    is_active: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)